pkgs/top-level/all-packages.nix

```diff
@@ -35617,7 +35617,9 @@ with pkgs;
     enableCli = false;
   };
   transmission_4-gtk = transmission_4.override { enableGTK3 = true; };
-  transmission_4-qt = transmission_4.override { enableQt = true; };
+  transmission_4-qt5 = transmission_4.override { enableQt5 = true; };
+  transmission_4-qt6 = transmission_4.override { enableQt6 = true; };
+  transmission_4-qt = transmission_4-qt5;
```

Comment:
I think that the default version should be the qt6 one, that's what they are shipping by default on windows and mac in their official builds, I see no reason to diverge intentionally.

Reply:
The reason is that plasma6 only landed *very* recently in unstable, so we are likely to have a number of people who are still using plasma5 and without the plasma6 themes installed, the qt6 version will look out of place. We have been carrying a few applications in both their qt5 and qt6 versions since before plasma6 landed and they all stick to qt5 by default to avoid them looking bad.

Reply:
We can always toggle the default later, so I'm going to go ahead and merge this.